fastlane/spec/actions_specs/import_from_git_spec.rb

```diff
@@ -208,26 +209,53 @@ def self.is_supported?(platform)
           end").runner.execute(:test)
         end
 
-        # Because we don't know if the specified value represents a branch or a
-        # tag. And knowing that it doesn't make sense to cache a branch, that
-        # points to a different commit when one is added, as opposed to a tag,
-        # we decided to enable caching only when used with the `version` param.
-        it "ignores it when tag is specified via branch param" do
-          expect(UI).not_to receive(:message).with(caching_message)
-          expect(UI).to receive(:important).with('Works since v1')
+        # Meaning, `git fetch` and `git merge` are performed when caching is eligible
+        # and the version isn't specified in the local clone.
```

Comment:
What do you mean by "in the local clone"?

Reply:
I meant the repo that you imported in the cache path, but I think the version should be specified in `import_git_from` action so it can be removed.